serverless/installing-kn.adoc

```diff
@@ -20,3 +20,9 @@ include::modules/serverless-installing-cli-linux-rpm.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-linux.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-macos.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-windows.adoc[leveloffset=+1]
+include::modules/serverless-kn-cli-plugins.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Think this should be a H2 heading?

Reply:
The `.` markup seems to be used twice as much as the `== ` one in the whole `openshift-docs`, but I did go with `== ` to comply with how you do it in `openshift-docs/serverless`.